tests/integration-service/namespace-backed-environments.go

```diff
@@ -123,32 +125,27 @@ var _ = framework.IntegrationServiceSuiteDescribe("Namespace-backed Environment
 		It("checks for deploymentTargetClaim after Ephemeral env has been created", func() {
 			Eventually(func() error {
 				dtcl, err = f.AsKubeDeveloper.GitOpsController.GetDeploymentTargetClaimsList(testNamespace)
-				if dtcl.Items == nil {
-					return fmt.Errorf("DeploymentTargetClaim is nil.")
-				}
-				if dtcl.Items[0].Status.Phase != phaseDTC {
-					return fmt.Errorf("DeploymentTargetClaimPhase is: " + string(dtcl.Items[0].Status.Phase) + " Expected: " + string(phaseDTC))
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
I think we still need to check for `len(dtcl.Items) > 0` here and in other places  (below) as well, because I think there could be a case when `GetDeploymentTargetClaimsList` func returns no error, but the `dctl.Items` is empty

Reply:
Alright let me add it.